src/activities/quizzes/QuizEntity.js

```diff
@@ -800,7 +800,12 @@ export class QuizEntity extends Entity {
 	async checkin() {
 		if (this.canCheckin()) {
 			const action = this.getActionByName(Actions.workingCopy.checkin);
-			const entity = await performSirenAction(this._token, action);
+			let entity;
+			try {
+				entity = await performSirenAction(this._token, action);
+			} catch (e) {
+				return Promise.reject(e);
```

Comment:
How are we handling the rejected promise? Or do all rejected promises get caught somewhere?

Checkin is being called [here ](https://search.d2l.dev/xref/BrightspaceHypermediaComponents/activities/components/d2l-activity-editor/mixins/d2l-activity-editor-working-copy-dialog-mixin.js?r=cc8284f0#53) and [here ](https://search.d2l.dev/xref/BrightspaceHypermediaComponents/activities/components/d2l-activity-editor/mixins/d2l-activity-editor-working-copy-dialog-mixin.js?r=cc8284f0#53) right?

Reply:
In this PR: [here] (https://github.com/BrightspaceHypermediaComponents/activities/pull/1576/files#diff-c1373be40678baae8550c911ebc996af36e660be5cebe234851576a9bbee6127R29) and [then here] (https://github.com/BrightspaceHypermediaComponents/activities/pull/1576/files#diff-d91a72513852488e81d1552fe1c9d5209e498f6b5f099407448e697765db7badR60)

Reply:
Oh...lets all pretend that didn't happen. Cool. Makes sense to me.